target/linux/airoha/files/include/linux/mfd/airoha-en7581-mfd.h

```diff
@@ -0,0 +1,10 @@
+/* SPDX-License-Identifier: GPL-2.0 */
+#ifndef _LINUX_INCLUDE_MFD_AIROHA_EN7581_MFD_H_
+#define _LINUX_INCLUDE_MFD_AIROHA_EN7581_MFD_H_
+
+struct airoha_mfd {
+	void __iomem *base;
+	struct regmap *chip_scu;
```

Comment:
@Ansuel this is not necessary I would say, we can do the lookup in the pinctrl driver